ui/admin/app/components/form/host-catalog/aws/index.hbs

```diff
@@ -101,14 +101,9 @@
       </:body>
     </form.fieldset>
   {{else}}
-    <InfoField
-      @value={{t 'resources.host-catalog.types.aws'}}
-      @icon='aws-color'
-      disabled={{form.disabled}}
-      as |F|
-    >
+    <InfoField @value={{t 'descriptions.provider'}} @icon='aws-color' as |F|>
```

Comment:
I'm a bit confused, wasn't the `descriptions.provider` previously the helper text before we migrated to `InfoField`? Why is it now the value instead of `Azure` or `AWS`?

Reply:
@DhariniJeeva yes that is correct.  We use a `hds::form::textinput::field`  that has better accessibility that what we previously had it.

Reply:
The component itself has the elements reversed, which caused the confusion.  That will be fixed in a separate PR.  The value should be the value.  The helper text should be the description/help/explanation.

Reply:
I'll close this pr as this is fixed in https://github.com/hashicorp/boundary-ui/pull/1525